Do not panic if network is nil

vdemeester opened the pull request:
Fixes #28633.

network is `nil` if the following case:

```
services:
  foo:
    image: nginx
    networks:
      mynetwork:
```

It's a valid compose so we should not panic.

/cc @aanand @dnephin 

:frog: 

Signed-off-by: Vincent Demeester <vincent@sbr.pm>

dnephin commented on cli/command/stack/deploy.go:
```diff
@@ -252,9 +252,13 @@ func convertServiceNetworks(
+		var aliases []string
+		if network != nil {
+			aliases = append(network.Aliases, name)
+		}
```
We always want to add the `name` alias, so I think this needs to be `if network == nil { // initialize network }`

vdemeester replied:
Ah you are right :+1:

vdemeester committed:
Do not panic if network is nil

network is `nil` if the following case:

```
services:
  foo:
    image: nginx
    networks:
      mynetwork:
```

It's a valid compose so we should not panic.

Signed-off-by: Vincent Demeester <vincent@sbr.pm>

crosbymichael commented:
LGTM

runcom commented:
> It's a valid compose so we should not panic.

that leading `:` in your example above is misleading there though :confused:

tiborvass commented:
LGTM

tiborvass commented:
Failure is authz plugin, unrelated.

vieux commented:
LGTM
